### PR TITLE
Feature/splash screen

### PR DESCRIPTION
**Pull request by JonathanFager**
#### splashscreen.java

A new javaclass that uses a thread and puts it to sleep for the duration of splashscreen, then launces mainactivity and destoyrs the splashscreen
#### splash.xml

layout for splashscreen
#### anim folder res/anim

a folder for animation xmls.
#### busify_logo1

logo, not used at this time
#### busify_logo_white

logo, used for splashscreen
#### AndroidManifest

Changed intent to launch splashscreen.java

### CODE REVIEW

**Comment by dgenelov** [comment]
Well done! :)
A few remarks.
- [x] @author and @version parameters missing in SplashScreen.java javadoc
- [x] Spelling errors in onPause() comments and in class description javadoc
- [x] View doesn't fade out. Instead, it just changes to mainActivity instantly. Update this if possible.

**Comment by dgenelov** [comment]
Spelling errors in onPause() comments

**Comment by dgenelov** [comment]
View doesn't fade out. Instead, it just changes to mainActivity instantly. Update this if possible.

**Comment by melindaflp** [comment]
- [ ]

File: app/src/main/java/com/example/eliasvensson/busify/SplashScreen.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                    startActivity(intent);
+                }
+            }
+        };
+        timerThread.start();
+    }
+
+    @Override
+    protected void onPause() {
+        //onPause is run when activity switches. Destoyes the thread and therefor the splashscreen
```
Comment (RegularSvensson) [inline app/src/main/java/com/example/eliasvensson/busify/SplashScreen.java]: Spelling error

**Pull request merged**